spec/base_model_spec.rb

```diff
@@ -0,0 +1,55 @@
+describe AnsibleTowerClient::BaseModel do
+  let(:api) { double("api") }
+  let(:hash) do
+    {
+      "name"            => "jeff",
+      "address"         => { "street" => "22 charlotte rd"},
+      "serialized_json" => { "key" => "value"}.to_json,
+      "serialized_yaml" => { "key" => "value"}.to_yaml
+    }
+  end
+
+  context ".initialize" do
+    shared_examples_for "validate new object" do
```

Comment:
I like how this is laid out. Easy to read in the context of the other tests in this file.